packages/perseus-core/README.md

```diff
@@ -0,0 +1,3 @@
+# @khanacademy/perseus-core
```

Comment:
We discussed naming this simply: `perseus` in https://khanacademy.atlassian.net/wiki/spaces/ENG/pages/1960804402/Perseus+Repo+Organization

Reply:
I guess this wasn't actually all that difficult.